Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -5,11 +5,11 @@
 import StreamChat
 import UIKit
 
-/// A `ChatChannelSwipeableListItemView` subclass view that shows channel information.
+/// A `SwipeableView` subclass view that shows channel information.
 public typealias ChatChannelListItemView = _ChatChannelListItemView<NoExtraData>
```

Comment:
`ChatChannelListItemView` is not a `SwipeableView` subclass

Reply:
added.